pom.xml

```diff
@@ -80,6 +80,9 @@
         <version.server.bom>24.0.0.Beta1-SNAPSHOT</version.server.bom>
         <version.microprofile.bom>24.0.0.Beta1-SNAPSHOT</version.microprofile.bom>
 
+        <version.server.bootable-jar>23.0.0.Final</version.server.bootable-jar>
```

Comment:
Any reason for value different from the properties just above?

Reply:
yes: the quickstart relies on snapshot of WildFly server but on release of the Bootable Jar plug-in.

(e.g. https://github.com/wildfly/quickstart/blob/52fd07d85208df512c7f0a31d20f3fd614f9d58f/microprofile-config/pom.xml#L23)

I'm moving this property to the root `pom.xml` to avoid having to repeat that in all quickstarts that use bootable jar.